README.md

```diff
@@ -582,6 +582,7 @@ Libraries to help manage database schemas and migrations.
 ## Filtering and Validation
 *Libraries for filtering and validating data.*
 
+* [HTMLPurifier](http://htmlpurifier.org/) - A HTML filtering library.
```

Comment:
GitHub repo provide source code at first, `composer.json` for install via composer that's defacto PHP standart library installation for now, and the link to the official docs website. I vote :+1: for GitHub link